src/server/worldserver/IRCBridge/IRCBridge.cpp

```diff
@@ -256,6 +257,7 @@ void IRCBridge::Login()
 
 std::string IRCBridge::MakeStringUpper(std::string string)
 {
-    std::transform(string.begin(), string.end(), string.begin(), ::toupper);
-    return string;
+    std::string temp = string;
+    boost::to_upper(temp);
+    return temp;
```

Comment:
there is no return value, so I have to copy again

and boost over std::transform because its more efficient and has less troubles with special chars